core/arch/arm/kernel/thread.c

```diff
@@ -975,10 +1013,10 @@ static vaddr_t get_excp_vect(void)
 	case CORTEX_A72_PART_NUM:
 	case CORTEX_A73_PART_NUM:
 	case CORTEX_A75_PART_NUM:
```

Comment:
Since we rely on ARM-TF for implementing the workaround, should we still keep this list of affected cores?

Reply:
Yes, I see no harm in that.